src/styles/StyledElements.ts

```diff
@@ -3,6 +3,17 @@ import color from './colors';
 import fonts from './fonts';
 import resetCSS from './reset';
 
+export const H1Styled = styled.h1`
```

Comment:
Nice PR @BillyG83. Why you didn't use `<Typography>` instead of creating a new H1 style

Reply:
done TY, i'll leave this in for now and clean up all the `StyledElements` later if thats ok